pythia/modules/layers.py

```diff
@@ -273,6 +309,45 @@ def forward(self, image_feat, question_embedding, context_embedding=None):
         return joint_feature
 
 
+class TopDownAttentionLSTM(nn.Module):
```

Comment:
This should be moved to `attention.py`

Reply:
I have added `TopDownAttentionLSTM` as a type of `ModalCombineLayer` so that we can reuse the `TopDownAttention` module when we process the feature and word embeddings. Does it still make sense to move to `attention.py` as this is only a type of layer?

Reply:
Ok, keeping this here makes sense.